_dashboards/drag-drop-wizard.md

```diff
@@ -13,39 +13,52 @@ You can use the drag and drop visualization wizard in OpenSearch Dashboards to c
 
 The drag and drop visualization wizard:
```

Comment:
"drag-and-drop" should be hyphenated when used as a compound modifier. Please apply globally.